demos/button/icons.html

```diff
@@ -4,44 +4,59 @@
 	<meta charset="utf-8">
 	<title>jQuery UI Button - Icons</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/button.js"></script>
 	<link rel="stylesheet" href="../demos.css">
-	<script>
-	$(function() {
-		$( "button:first" ).button({
-			icons: {
-				primary: "ui-icon-locked"
-			},
-			text: false
-		}).next().button({
-			icons: {
-				primary: "ui-icon-locked"
-			}
-		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			}
-		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			},
-			text: false
-		});
-	});
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js">
+		$( ".widget button" )
+			.eq( 0 ).button()
+			.end().eq( 1 ).button( {
+				icon: "ui-icon-gear",
+				showLabel: false
+			} ).end().eq( 2 ).button( {
+				icon: "ui-icon-gear"
+			} ).end().eq( 3 ).button( {
+				icon: "ui-icon-gear",
+				iconPosition: "end"
+			} ).end().eq( 4 ).button( {
+				icon: "ui-icon-gear",
+				iconPosition: "top"
+			} ).end().eq( 5 ).button( {
+				icon: "ui-icon-gear",
+				iconPosition: "bottom"
+			} );
 	</script>
 </head>
 <body>
-
-<button>Button with icon only</button>
-<button>Button with icon on the left</button>
-<button>Button with two icons</button>
-<button>Button with two icons and no text</button>
-
+<div class="widget">
+	<h1>Widget</h1>
+	<button>Button with only text</button>
+	<button>Button with icon only</button>
+	<button>Button with icon on the left</button>
+	<button>Button with icon on the right</button>
+	<button>Button with icon on the top</button>
+	<button>Button with icon on the bottom</button>
+</div>
+<div class="css">
+	<h1>CSS</h1>
+	<button class="ui-button ui-widget ui-corner-all">
+		Button with only text
+	</button>
+	<button class="ui-button ui-widget ui-corner-all ui-button-icon-only" title="Button with icon only">
+		<span class="ui-icon ui-icon-gear"></span> Button with icon only
+	</button>
+	<button class="ui-button ui-widget ui-corner-all">
+		<span class="ui-icon ui-icon-gear"></span> Button with icon on the left
```

Comment:
Then this should be correct: https://github.com/jquery/api.jqueryui.com/pull/280/files#diff-bf14c9f5ccd16aa682040e25fcc9231eR19